command/commands.go

```diff
@@ -913,9 +916,24 @@ func initCommands(ui, serverCmdUi cli.Ui, runOpts *RunOptions) map[string]cli.Co
 	}
 
 	entInitCommands(ui, serverCmdUi, runOpts, commands)
+	initHCPCommands(ui, commands)
+
 	return commands
 }
 
+func initHCPCommands(ui cli.Ui, commands map[string]cli.CommandFactory) {
+	for cmd, cmdFactory := range hcpvlib.InitHCPCommand(ui) {
+		// check for conflicts and only put command in the map in case it doesn't conflict with existing one
```

Comment:
Seems like this comment is not completely aligned with the code. It seems on the first failure, we break from trying the rest.

Reply:
Out of curiosity, how does HCP lib know which commands not to introduce to avoid conflicts?

Reply:
We don't add commands through the lib, we scan the commands the lib makes available and add them. The conflicts are checked in the vault main's logic.